Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -568,6 +544,60 @@ public void timeUpdateFlow(String time, String lat, String lng) {
         countAppUsageDays();
     }
 
+    public void checkUpdate(){
+        SharedPreferences sharedPref = this.getSharedPreferences(
```

Comment:
move this function to `CommonJsInterface` as we will not have access to mainActivity after dependency update. send activity and context as input to this function

Reply:
Done